Update Radius comparison (#925)
radius-project/docs: docs/content/concepts/faq/index.md

[Tanzu](https://tanzu.vmware.com/) is a portfolio of products and services for modernizing applications and infrastructure with a common goal: deliver better software to production, continuously.

> Radius is an open-source project that provides a platform for modeling and deploying applications. Radius is not a paid product or service. Instead, Radius allows you to host your own Radius environments on your own Kubernetes clusters, and deploy applications to any cloud provider or on-premises infrastructure. Radius is providing new value is through Recipes and the Application Graph. Recipes make it easier for developers and platform engineers/IT engineers to collaborate with each other in building cloud native apps. The Application Graph makes it easier to see all of the various components that make up a cloud native application. We think these are additive and unique.

Reviewer: ```suggestion
Radius is an open-source project that provides a platform for modeling and deploying applications. Radius is not a paid product or service. Instead, Radius allows you to host your own Radius environments on your own Kubernetes clusters, and deploy applications to any cloud provider or on-premises infrastructure. Radius provides differentiated value through Recipes and the Application Graph. Recipes make it easier for developers and platform engineers/IT engineers to collaborate when building cloud native apps. The Application Graph makes it easier to see all of the various components that make up a cloud native application.
```
